src/components/Form/FormProvider.js

```diff
@@ -260,6 +260,22 @@ const FormProvider = forwardRef(
                     inputValues[inputID] = _.isUndefined(propsToParse.defaultValue) ? getInitialValueByType(propsToParse.valueType) : propsToParse.defaultValue;
                 }
 
+                // If the input is a submit editing input, we need to set the onSubmitEditing prop
+                // to the submit function of the form
+                const onSubmitEditingObject = shouldSubmitEdit
+                    ? {
+                          onSubmitEditing: (e) => {
+                              submit();
+                              if (!propsToParse.onSubmitEditing) {
+                                  return;
+                              }
+                              propsToParse.onSubmitEditing(e);
+                          },
+                      }
```

Comment:
Would you add `returnKeyType` to this object?

`onSubmitEditingObject` could be renamed to `formSubmitObject`.

We handle this two things in an inconsistent manner.